bears/general/SpaceConsistencyBear.py

```diff
@@ -38,48 +41,84 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
+        def get_blanklines_nr_start():
+
```

Comment:
though not violating PEP8, I usually prefer no blanklines between a `def` and code, this kinda interrupts reading flow imo. Optional though if you see it differently ;)